CHANGES.rst

```diff
@@ -1,6 +1,12 @@
 Change Log
 ==========
 
+**Unreleased**
+
+* :meth:`Emoji.update` support updating an emoji's permissions
+* :meth:`Emoji.add` support optionally passing booleans to set an emoji's
```

Comment:
This method is `SubredditEmoji.add`, not `Emoji.add`. For proper Sphinx style, please make this read

    :meth:`~.SubredditEmoji.add`

In addition, please change "support" to "supports".